src/app/organization/sms-campaigns/sms-campaign-stepper/edit-sms-campaign-step/edit-business-rule-parameters/edit-business-rule-parameters.component.ts

```
@@ -88,19 +89,21 @@ export class EditBusinessRuleParametersComponent implements OnInit, OnChanges {
-            break;
+        const controlValue = this.paramValue[param.variable]?.toString();
+        if (controlValue) {
+          switch (param.displayType) {
+            case 'text':
+              this.ReportForm.get(param.name).patchValue(controlValue);
+              break;
+            case 'select':
+              this.fetchSelectOptions(param, param.name);
+              break;
+            case 'date':
+              const dateFormat = this.settingsService.dateFormat;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)